moebot_bot/bot/bot.go

```diff
@@ -148,6 +149,63 @@ func messageCreate(session *discordgo.Session, message *discordgo.MessageCreate)
 	//distributeTickets(guild, message, session, messageTime)
 }
 
+func messageReactionAdd(session *discordgo.Session, reactionAdd *discordgo.MessageReactionAdd) {
+	polls, err := db.PollsOpenQuery()
```

Comment:
Not sure how I feel about a forced db query for every reaction. That's not a big deal now but could get really painful on the DB later.
What if we instead add a check for if the message belongs to a bot (or moebot)? `if message.Author.Bot { ... ` since polls will always come from moebot. `session.State.User.ID` also has moebot's ID in it which we could use to compare. We can add in more robust `do this if bot, do this if user` to the reaction processing later.

Reply:
Now that I looked at it further we don't have the context of the message here... and a DB query sounds better than a network request... I'll think about this but for now it's no problem 👍

Reply:
The other solution would be to keep an in-memory list of polls, it would require us to slightly modify the bot's code, I'll do it in another branch and show you before merging